pfcp/adapter/adapter.go

```diff
@@ -39,7 +39,7 @@ func InsertPfcpTxn(seqNo uint32, upNodeID *pfcpType.NodeID) {
 	PfcpTxns[seqNo] = upNodeID
 }
 
-func HandleAdapterPfcpRsp(pfcpMsg pfcp.Message) error {
+func HandleAdapterPfcpRsp(pfcpMsg pfcp.Message, evtData *pfcpUdp.PfcpEventData) error {
```

Comment:
Comment at start..why we don't need lock on upf in this function. Assumption is that this is synchronous call and called is already holding the upf lock ?